### DISCUSSION TITLE
Split stem notation

### DISCUSSION THREAD

**Discussion by GBogdahn** [w3c-cg/musicxml#616] [Q&A] [answered]
Until now it's impossible to encode split stem notation.
![image](https://user-images.githubusercontent.com/64458816/109953813-d3a11580-7ce0-11eb-8947-6d6e5b7ee8a4.png)

Notation programs offer some workarounds but for some cases it's important to have split stems:
![image](https://user-images.githubusercontent.com/64458816/109953999-14009380-7ce1-11eb-8369-6c1b7947e41f.png)

There is also an issue for making these possible in MEI
https://github.com/music-encoding/music-encoding/issues/721

**Comment by mdgood** [comment, marked as answer]
I believe that MusicXML can already do this by specifying different default-x values for the different `<note>` elements within a chord, and specifying both default-x and default-y values for the `<stem>` element on the split stem note. Usually notes only specify a default-y value for regular stems. I don't know of any software that is supporting this yet though.

@GBogdahn, if I am misunderstanding something please let me know.
